configure

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+rm -rf CMakeFiles/ \
+       cmake_install.cmake \
+       CMakeCache.txt
+
+#$(psi4 --plugin-compile)
+cmake -C /home/sina/MySoftwares/psi4_01_09_2020/objdir/stage/share/cmake/psi4/psi4PluginCache.cmake \
+      -D CMAKE_PREFIX_PATH=/home/sina/MySoftwares/psi4_01_09_2020/objdir/stage\
+      -D ENABLE_OPENRDM=ON \
+      -D openrdm_DIR="/home/sina/MOLSSI/RDMInoles-Prototype/build/stage/share/cmake/openrdm" \
+      .
+
+cmake --build . -- -j $(nproc)
```

Comment:
did you mean for file this to be included?

Reply:
Please read the comment below.